tests/websockets/test_graphql_ws.py

```diff
@@ -80,7 +80,6 @@ async def test_simple_subscription(ws: WebSocketClient):
     [
         ({}, "Hi1"),
         ({"operationName": None}, "Hi1"),
-        ({"operationName": ""}, "Hi1"),
```

Comment:
This is the WS test case I had to fix. An empty string as an operation name is valid, and we should try to look it up in the document.